Cargo.toml

```diff
@@ -80,6 +83,9 @@ git = "https://github.com/pingcap/rust-prometheus.git"
 default-features = false
 features = ["nightly", "push"]
 
+[dependencies.grpc]
+git = "https://github.com/pingcap/grpc-rust.git"
```

Comment:
No, we don't, for now, `pingcap/grpc-rust` is an update-to-date mirror of upstream.